cmd/chainvet/doc.go

```diff
@@ -5,26 +5,26 @@
 
 /*
 
-Vet examines Go source code and reports suspicious constructs, such as Printf
-calls whose arguments do not align with the format string. Vet uses heuristics
-that do not guarantee all reports are genuine problems, but it can find errors
-not caught by the compilers.
+chainvet examines Go source code and reports suspicious constructs, such as
+Printf calls whose arguments do not align with the format string. chainvet
+uses heuristics that do not guarantee all reports are genuine problems,
+but it can find errors not caught by the compilers.
 
 It can be invoked three ways:
 
 By package, from the go tool:
-	go vet package/path/name
+	chainvet package/path/name
```

Comment:
Ooo I bet this way doesn't work at all, because `vet` relies on the go tool to find all the package files for it. Our chainvet is a fork of `vet` so I think it only supports the second two forms.

And there's no way to teach `go` how to invoke chainvet in this way. Even if we dropped our binary in the tools dir, that would let us run it as `go tool chainvet` but that's no better than just running `chainvet` directly.